src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ImportStatusBau.cs

```diff
@@ -1,18 +1,18 @@
 using System;
-using GovUk.Education.ExploreEducationStatistics.Common.Services;
 using GovUk.Education.ExploreEducationStatistics.Data.Processor.Model;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
 {
     public class ImportStatusBau
     {
+        public string SubjectTitle { get; set; }
         public Guid SubjectId { get; set; }
+        public Guid PublicationId { get; set; }
+        public Guid ReleaseId { get; set; }
         public string DataFileName { get; set; }
         public string MetaFileName { get; set; }
-        public Release Release { get; set; }
-        public string Errors { get; set; }
         public int NumberOfRows { get; set; }
-        public IStatus Status { get; set; }
+        public string Status { get; set; }
```

Comment:
Rather than change this to an string (which is less safe), think you can add the following annotation and the JSON converter will do the rest:

```javascript
[JsonConverter(typeof(StringEnumConverter))]
public IStatus Status { get; set; }
```